discord/command.go

```diff
@@ -328,6 +329,7 @@ const (
 //    - *RoleOption
 //    - *MentionableOption
 //    - *NumberOption
+//	  - *AttachmentOption
```

Comment:
Seems to be a missing space here.

Reply:
Ah I think it's just that I used a tab and then 2 spaces, I'll swap it for spaces.